src/containerapp/azext_containerapp/commands.py

```diff
@@ -178,6 +178,12 @@ def load_command_table(self, _):
         g.custom_command('remove', 'remove_ip_restriction')
         g.custom_show_command('list', 'show_ip_restrictions')
 
+    with self.command_group('containerapp ingress cors') as g:
+        g.custom_command('enable', 'enable_cors_policy', exception_handler=ex_handler_factory())
```

Comment:
I see you have no_wait argument in function. If you want to support no wait, please add `supports_no_wait=True` and add wait command to help polling.

Reply:
Do not plan to support no-wait